packages/subgraph/src/dao/utils.ts

```diff
@@ -136,21 +142,32 @@ function createWhitelistVotingPackage(who: Address, daoId: string): void {
 }
 
 export function addPackage(daoId: string, who: Address): void {
-  let daoPackageEntityId = daoId + '_' + who.toHexString();
-  let daoPackageEntity = new DaoPackage(daoPackageEntityId);
-  daoPackageEntity.pkg = who.toHexString();
-  daoPackageEntity.dao = daoId;
-  daoPackageEntity.save();
-
   // package
-  // @dev this is a temporary solution as we have only 2 packages, and should change in the future.
-  let contract = ERC20VotingContract.bind(who);
-  let response = contract.try_token();
-  if (!response.reverted) {
+  // TODO: rethink this once the market place is ready
+  let contract = ERC165Contract.bind(who);
```

Comment:
use `const` instead of `let` if it is not planned to be overwritten

Reply:
agreed, but i just wanted to follow the other subgraph examples